pkg/synopsysctl/cmd_create.go

```diff
@@ -663,9 +670,33 @@ func updatePolarisSpecWithFlags(cmd *cobra.Command, namespace string) (*polaris.
 	}
 	polarisSpec.Namespace = namespace
 
+	if err := validatePolaris(polarisSpec); err != nil {
+		return nil, err
+	}
 	return &polarisSpec, nil
 }
 
+func validatePolaris(polarisConf polaris.Polaris) error {
```

Comment:
Can you pls add all the validation in Args method?

Args: func(cmd *cobra.Command, args []string) error {
		// Check the Number of Arguments
		if len(args) != 1 {
			cmd.Help()
			return fmt.Errorf("this command takes 1 argument")
		}
		checkPasswords(cmd.Flags())
		return nil
	},